index.html

```diff
@@ -13,18 +13,18 @@
       </ul>
     </div>
     <script>    
-      var ul = document.getElementById('js-list');
+      const ul = document.getElementById('js-list');
       for(var i=1; i<=2; i++) {
-      var li = document.createElement('li');
+      const li = document.createElement('li');
       li.textContent = 'a'+i;
```

Comment:
[IMO].
Currently, the text is added to the li tag, but that doesn't include the text in the a tag, so add the text to a instead of li.

Reply:
Thank you for your comment. Move the text.